### PR TITLE
[WIP] add captions to partha-git-addCaption

### PR DESCRIPTION
**Pull request by partha-git**
# Its a big header

### CODE REVIEW

**Review by dmudigonda — APPROVED** [review]
This looks good

**Review by mithunganguly — COMMENTED** [review]
Looks fine

**Review by mithunganguly — APPROVED** [review]
Looks good. Approved

File: _posts/0000-01-12-partha-git.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![philosoraptor](https://cloud.githubusercontent.com/assets/16547949/25401001/66d677c6-29c2-11e7-9b12-263e036c236e.jpg)
 {: .center}
 
-Caption here
+Dinosaur trying to understand string theory :p
```
Comment (mithunganguly) [inline _posts/0000-01-12-partha-git.md:9]: good

Comment (mithunganguly) [reply _posts/0000-01-12-partha-git.md:9]: Looks good

**Review by mithunganguly — COMMENTED** [review]
Reviewed

**Pull request merged**